uaclient/status.py

```diff
@@ -132,9 +133,19 @@ class UserFacingStatus(enum.Enum):
 This machine is now attached to '{contract_name}'.
 """
 
-MESSAGE_ENABLE_BY_DEFAULT_TMPL = "Enabling default service {name}"
+MESSAGE_CONTRACT_EXPIRED_ERROR = """\
+Subscription has expired
+To obtain a token please visit: https://ubuntu.com/advantage"""
+MESSAGE_INVALID_SERVICE_OP_FAILURE_TMPL = """\
+Cannot {operation} ‘{name}’
```

Comment:
Can we use standard quote characters here, or have we been asked to ~~make piping our output really annoying~~ (:wink:) use these quotes?